datasimulator/file_handling.py

```diff
@@ -0,0 +1,11 @@
+from cdislogging import get_logger
+logger = get_logger("data-simulator", log_level="info")
+
+
+def write_submission_order_to_file(submission_order, file_path):
+    """ given a correct file path, writes a name-category pair to the file from a list of nodes (submissions) """
```

Comment:
```suggestion
    """ given a correct file path, writes a name-category pair to the file from a list of nodes """
```
Not sure what you mean by submissions here, but this is at the node level, not at the submission record level